tests/integration/go_ethereum/test_goethereum_http.py

```diff
@@ -70,7 +71,7 @@ def w3(geth_process, endpoint_uri):
     return Web3(Web3.HTTPProvider(endpoint_uri))
 
 
-class TestGoEthereumTest(GoEthereumTest):
+class TestGoEthereumWeb3ModuleTest(GoEthereumWeb3ModuleTest):
```

Comment:
There's a lot of this renaming in the integration tests. Is it just cleaning up for clarity, or more relevant in a way I'm not seeing?

Reply:
The biggest change is making a proper `AsyncWeb3ModuleTest` and then yeah the renaming is for clarity. The old `Web3ModuleTest` was checking all the static utility methods for conversions, etc. Since we have some actual async methods on the `AsyncWeb3` class directly now, this PR adds a separation of logic between sync and async and gets rid of the `@pytest.mark.parametrize` that cycled between `Web3` and `AsyncWeb3`, opting for two separate test classes instead.